nautobot/extras/tests/test_models.py

```diff
@@ -717,6 +717,10 @@ def setUp(self):
         self.test_file = SimpleUploadedFile(name="test_file.txt", content=b"I am content.\n")
         self.fp = FileProxy.objects.create(name=self.test_file.name, file=self.test_file)
 
+    @skip
```

Comment:
`@unittest.expectedFailure` perhaps?

Reply:
I think to use that I'd have to then do a `super()` call so as to make sure the test actually runs as such? Not sure it's worth it personally but happy to make the change if you think it'd be clearer.